draft-ietf-rats-corim.md

```diff
@@ -795,25 +795,43 @@ the `authorized-by` statement can be supplied in the `measurement-map`.
 {::include cddl/measurement-map.cddl}
 ~~~
 
+where `mkeyvalue-pair` is
+
+~~~ cddl
+{::include cddl/mkeyvalue-pair.cddl}
+~~~
+
 The following describes each member of the `measurement-map`:
 
-* `mkey` (index 0): An optional unique identifier of the measured
-  (sub-)environment.  See {{sec-comid-mkey}}.
+* `mkey` (index 0): DEPRECATED (remove for final RFC) An optional identifier of the measured element.
```

Comment:
I would clarify this in a better way....

I guess you intend to remove the index 0 for `mkey` but want to retain the `mkey` for your mkeyvalue-pair